scripts/install-telegraf.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/sh
```

Comment:
This should be "#!/bin/sh" (or #!/bin/bash) - it's not valid on my wsl2 or ubuntu linux.

Reply:
update done